rebuild/src/secret_key.h

```diff
@@ -0,0 +1,28 @@
+#ifndef BLST_TS_SECRET_KEY_H__
+#define BLST_TS_SECRET_KEY_H__
+
+#include <memory>
+
+#include "addon.h"
+#include "blst.hpp"
+#include "napi.h"
+#include "public_key.h"
+#include "signature.h"
```

Comment:
Are all those required? I'd limit the number of includes in header files to speed up compilation and make the dependencies simpler. Check out https://github.com/include-what-you-use/include-what-you-use

Reply:
updated in 560e699492ee1c45e3d6c6e67a65a475a8a295ff